### PR TITLE
ReversingLabs basic playbooks

### PR DESCRIPTION
**Pull request by ivukovicRL**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added 4 playbooks, fixed minor bug

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by daryakoval** [comment]
@ivukovicRL Please fill the [Contribution Registration form](https://forms.gle/XDfxU4E61ZwEESSMA).

**Comment by ivukovicRL** [comment]
> @ivukovicRL Please see my comments [here](https://github.com/demisto/content/pull/15267#pullrequestreview-776173712).
> 
> After you apply all my changes lets schedule a demo session.

Hi, thanks for the review! Sure for the demo, is sometimes tomorrow 08-17(CET) ok for you?

File: Packs/ReversingLabs_A1000/pack_metadata.json [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
     "description": "Powerful threat detection and file analysis platform. Get detailed information on each file's status and threat capabilities.",
     "support": "partner",
-    "currentVersion": "2.0.2",
+    "currentVersion": "2.0.3",
     "author": "ReversingLabs",
     "url": "https://www.reversinglabs.com/products/malware-threat-hunting-and-investigations",
     "email": "support@reversinglabs.com",
     "devEmail": "integrations-team@reversinglabs.com",
-    "created": "2020-04-14T00:00:00Z",
+    "created": "2020-10-08T00:00:00Z",
```
Comment (daryakoval) [inline Packs/ReversingLabs_A1000/pack_metadata.json:10]: Why you changing the creation date?

File: Packs/ReversingLabs_TitaniumScale/pack_metadata.json [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
     "description": "Extract internal threat indicators with static malware analysis engine. Classify files and determine threat level.",
     "support": "partner",
-    "currentVersion": "1.0.0",
+    "currentVersion": "1.0.1",
     "author": "ReversingLabs",
     "url": "https://www.reversinglabs.com/products/malware-threat-hunting-and-investigations",
     "email": "support@reversinglabs.com",
     "devEmail": "integrations-team@reversinglabs.com",
-    "created": "2021-06-14T00:00:00Z",
+    "created": "2021-10-08T00:00:00Z",
```
Comment (daryakoval) [inline Packs/ReversingLabs_TitaniumScale/pack_metadata.json:10]: Same here, why this change is required?

File: Packs/ReversingLabs_Titanium_Cloud/pack_metadata.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
         "MislavReversingLabs",
         "ivukovicRL"
     ],
-  "created": "2020-04-14T00:00:00Z",
+  "created": "2020-10-08T00:00:00Z",
```
Comment (daryakoval) [inline Packs/ReversingLabs_Titanium_Cloud/pack_metadata.json:14]: Please do not change creation date.

**Review by daryakoval — CHANGES_REQUESTED** [review]

File: Packs/ReversingLabs_TitaniumScale/Playbooks/Detonate_File_-_ReversingLabs_TitaniumScale.yml [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+  description: The indicator type.
+- contextPath: DBotScore.Indicator
+  description: The indicator that was tested.
+- contextPath: DBotScore.Vendor
+  description: The vendor used to calculate the score.
+- contextPath: ReversingLabs.tc_report
+  description: Full report
+tests:
+- No tests (auto formatted)
+fromversion: 5.0.0
```
Comment (evisochek) [inline Packs/ReversingLabs_TitaniumScale/Playbooks/Detonate_File_-_ReversingLabs_TitaniumScale.yml:237]: If there is no prevention, Please update to 6.0.0

File: Packs/ReversingLabs_Titanium_Cloud/Playbooks/File_Reputation_-_ReversingLabs_TitaniumCloud.yml [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+  description: The indicator type.
+- contextPath: DBotScore.Indicator
+  description: The indicator that was tested.
+- contextPath: DBotScore.Vendor
+  description: The vendor used to calculate the score.
+- contextPath: ReversingLabs.file_reputation
+  description: Report in JSON.
+tests:
+- ReversingLabsTCTest
+fromversion: 5.0.0
```
Comment (evisochek) [inline Packs/ReversingLabs_Titanium_Cloud/Playbooks/File_Reputation_-_ReversingLabs_TitaniumCloud.yml:231]: If there is no prevention, Please update to 6.0.0

File: Packs/ReversingLabs_TitaniumScale/Playbooks/Detonate_File_-_ReversingLabs_TitaniumScale.yml [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+- contextPath: DBotScore.Type
+  description: The indicator type.
+- contextPath: DBotScore.Indicator
+  description: The indicator that was tested.
+- contextPath: DBotScore.Vendor
+  description: The vendor used to calculate the score.
+- contextPath: ReversingLabs.tc_report
+  description: Full report
+tests:
+- No tests (auto formatted)
```
[the remark this answers is not in the document]
Comment (evisochek) [reply Packs/ReversingLabs_TitaniumScale/Playbooks/Detonate_File_-_ReversingLabs_TitaniumScale.yml:236]: I think that those playbooks are related to the integration, I think that we should remove them.

File: Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py [hunk 7]
```diff
@@ -662,41 +662,45 @@ def url_report_output(response_json, url):
...
-
-    sources_table = tableToMarkdown("Third party sources", third_party_sources)
-    markdown = f"{markdown}\n {sources_table}"
+        markdown += f"""\n ### Third party statistics\n **TOTAL**: {third_party_statistics.get("total")}
+        **MALICIOUS**: {third_party_statistics.get("malicious")}
+        **CLEAN**: {third_party_statistics.get("clean")}
+        **UNDETECTED**: {third_party_statistics.get("undetected")}\n"""
+
+        sources_table = tableToMarkdown("Third party sources", third_party_sources)
+        markdown = f"{markdown}\n {sources_table}"
```
Comment (daryakoval) [inline Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py:703]: Please update Integration README with new human readable output for this command..

**Review by daryakoval — APPROVED** [review]

**Pull request merged**